x-pack/platform/plugins/shared/dataset_quality/public/components/dataset_quality/header.tsx

```diff
@@ -25,7 +24,7 @@ export default function Header() {
   } = useKibanaContextForPlugin();
   const { capabilities } = application;
 
-  const [ruleType, setRuleType] = useState<typeof DEGRADED_DOCS_RULE_TYPE_ID | null>(null);
+  const [isAlertFlyoutOpen, setIsAlertFlyoutOpen] = useState(false);
```

Comment:
Not sure about this change, the idea behind storing the ruleType instead of a boolean is to able to easily extend the types of alerts supported by this component in the future